server/src/domain/Experience.ts

```diff
@@ -0,0 +1,35 @@
+import { Column, Entity } from "typeorm";
+
+import { BaseModel } from "../infrastructure/BaseModel";
+
+export enum employmentType {
+    FullTime,
+    PartTime,
+    SelfEmployed,
+    Freelance,
+    InterShip,
+    ApprenticesShip,
+    Seasonal
+}
+
+@Entity({})
+export class Experience extends BaseModel {
+    @Column({ length: 100, /* unique:true */ }) // Como referencia, Github en un titulo de repositorio acepta como maximo 100
+    title: string
+
+    @Column({
+        enum: employmentType,
```

Comment:
Given the capitalization on this type name, you are likely to do:

```suggestion
        enum: EmploymentType,
```